infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -282,6 +288,19 @@ private static void configureLogger(Map<String, Object> opts) {
         root.setLevel(level);
     }
 
+    static void setupMetrics(ServerContext serverContext) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651478)